docs/tutorials/deployment.md

```diff
@@ -6,64 +6,74 @@ A few basic concepts about this process:
 __"Export method"__ is how a Python model is fully serialized to a deployable format.
 We support the following export methods:
 
-* `tracing`: see [pytorch documentation](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html) to learn about it
-* `scripting`: see [pytorch documentation](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html) to learn about it
+* `tracing`: see [pytorch documentation](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html) to learn about it\
+This method produces a standardized ONNX graph that can be used by any ONNX runtime, such as \
+[ONNX Runtime](https://onnxruntime.ai/)
+* `scripting`: see [pytorch documentation](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html) to learn about it\
+This method produces a standardized ONNX graph that can be used by any ONNX runtime, such as \
+[ONNX Runtime](https://onnxruntime.ai/)
```

Comment:
This makes it sound like tracing/scripting only works with ONNX graph format.. please remove them here and relevant info below to "Format" or "Runtime" section if needed.